### PR TITLE
Testmap update

### PR DESCRIPTION
**Pull request by ktilkath**

### CODE REVIEW

File: test/testUpgradeMap/README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
 # Usage
 There are three triggers, one to set the desired upgrade, and one to reset the desired upgrade.
 Change the triggers to set and reset the desired upgrade. Write  `set` in chat (and enter) to set the upgrades in game, use `reset` to reset.
+
+***
+v.02 ~ktilkath
```
Comment (satanja) [inline test/testUpgradeMap/README.md]: Version numbers are not needed. Simply extend the README to reflect your changes. Author is visible from the commit, so you can omit that as well.

File: test/testUpgradeMap/README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+Modified testmap:
+* added computer player
+* added some targets
+* edited phoenix upgrade triggers to "set phoenix" and "reset phoenix"
+***
```
Comment (satanja) [inline test/testUpgradeMap/README.md]: The point of the test map is that you modify the upgrade triggers in the editor, and add units to the map directly. If you happen to need enemy units, you can add those as well. Could you explain your reasoning of those features?

File: README.md [hunk 3]
```diff
@@ -32,5 +30,5 @@ All the data to get started with creating a PeepMode map can be found here, i.e.
 ## Authors
 * Jcraigk (first author of the PeepMode maps, now inactive).
 * **[satanja](https://github.com/satanja)** - discord: PiVMaSTeR#2096
-
+* ktilkath
```
Comment (satanja) [inline README.md]: Consider leaving your discord tag so people can tag you easily in the discord server.

Comment (ktilkath) [reply README.md]: agreed, will correct

**Pull request closed without merging**